Sources/DataPipeline/Config/SDKConfigBuilder.swift

```diff
@@ -137,12 +168,22 @@ public class SDKConfigBuilder {
         return self
     }
 
+    @available(iOSApplicationExtension, unavailable)
```

Comment:
We need this because we are accessing `AutoTrackingScreenViews` in it. I don't think this is an issue as [initialize method is already annotated similar](https://github.com/customerio/customerio-ios/blob/082d21b70e829381d7f4bc2c76b99f3c070c1185/Sources/DataPipeline/CustomerIO.swift#L9). So customers may never need to build config outside the normal initialization of SDK.

Reply:
have you verified this change with Cocoapods FCM app? i remember last time either not having it was causing issue or having it. Not 100% sure.

Reply:
All CI checks are working, that too might help us giving confidence that it's working for Cocoapods FCM app 🤔